pkg/virt-controller/services/template_test.go

```diff
@@ -1916,47 +1917,42 @@ var _ = Describe("Template", func() {
 		})
 
 		Context("with sriov interface", func() {
+			getVMIWithSriovInterface := func(name, uid string) *v1.VirtualMachineInstance {
+				return &v1.VirtualMachineInstance{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: name, Namespace: "default", UID: types.UID(uid),
+					},
+					Spec: v1.VirtualMachineInstanceSpec{
+						Domain: v1.DomainSpec{
+							Devices: v1.Devices{
+								DisableHotplug: true,
+								Interfaces: []v1.Interface{
+									v1.Interface{
+										Name: "sriov-nic",
+										InterfaceBindingMethod: v1.InterfaceBindingMethod{
+											SRIOV: &v1.InterfaceSRIOV{},
+										},
+									},
+								},
+							},
+						},
+					},
+				}
+			}
```

Comment:
It is too hard to read this now.

Reply:
In a different commit I would also try to remove `DisableHotplug: true`, I do not understand how it is relevant.

Reply:
I changed this function so it will be easy to understand
Also removed DisableHotplug on different commit